gamestonk_terminal/menu.py

```diff
@@ -1,11 +1,22 @@
 import os
 
+from matplotlib import pyplot
+
 from prompt_toolkit import PromptSession
+from prompt_toolkit.eventloop.inputhook import set_eventloop_with_inputhook
 from prompt_toolkit.history import FileHistory
 
+
+def inputhook(inputhook_context):
+    while not inputhook_context.input_is_ready():
+        pyplot.pause(0.1)
+    return False
+
+
 history_file = os.path.join(os.path.expanduser("~"), ".gamestonk_terminal.his")
 
 try:
     session = PromptSession(history=FileHistory(history_file))
+    set_eventloop_with_inputhook(inputhook)
 except Exception as e:  # noqa: F841
-    session = None
```

Comment:
I think the removal of ```session = None``` breaks the tests.

Reply:
Yeah, this was what did it. I added a warning message here so that if this code path is taken (prompt_toolkit is configured but constructing the session fails) it'd be clear what was going on. Thanks for the tip.